test/hotspot/jtreg/containers/docker/TestMemoryAwareness.java

```diff
@@ -63,10 +63,15 @@ public static void main(String[] args) throws Exception {
         DockerTestUtils.buildJdkContainerImage(imageName);
 
         try {
-            testMemoryLimit("100m", "104857600");
-            testMemoryLimit("500m", "524288000");
-            testMemoryLimit("1g", "1073741824");
-            testMemoryLimit("4g", "4294967296");
+            testMemoryLimit("100m", "104857600", false);
+            testMemoryLimit("500m", "524288000", false);
+            testMemoryLimit("1g", "1073741824", false);
+            testMemoryLimit("4g", "4294967296", false);
+            testMemoryLimit("100m", "104857600", true);
+            testMemoryLimit("500m", "524288000", true);
+            testMemoryLimit("1g", "1073741824", true);
+            testMemoryLimit("4g", "4294967296", true);
```

Comment:
One test with the additional cgroup mounts should be sufficient. I suggest:

```
testMemoryLimit("500m", "524288000", true /* additional cgroup mount */);
```

Reply:
done